### ISSUE TITLE
Stay Focused on The Currently Selected Page

### ISSUE THREAD

**Issue by khajavi** [zio/zio#8331]
Following [this](https://discord.com/channels/629491597070827530/629491597070827534/1137373743195365516) thread on Discord, it is annoying that the items on the ecosystem sidebar cannot stay focused. So we need to find out a way to fix that problem.

One solution is to make a separate sidebar for each project, but the problem is that we can't change/customize the breadcrumbs to show the hierarchy of documents.

Update: I paused working on this issue until we find a solution that doesn't reduce the accessibility (e.g. easy to navigate between ecosystem docs).

**Comment by hearnadam** [comment]
@khajavi is this done? it seems to work?

**Comment by khajavi** [comment]
@hearnadam No! I expect this should be fixed on the respective Docusaurus documentation framework!
I've to check, and see if this is reported there, or fixed in recent versions!
